### PR TITLE
CB-4140 RequestId propagation should work accross CB

### PR DESCRIPTION
**Pull request by horadla23**

### CODE REVIEW

File: datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java [hunk 1]
```diff
@@ -55,20 +55,17 @@
...
-        String requestId = threadBasedRequestIdProvider.getRequestId();
+        String requestId = MDCBuilder.getRequestId().get();
         notify(selector, new SdxEvent(selector, sdxId, userId, requestId));
     }
 
     public void triggerSdxDeletion(Long sdxId, boolean forced) {
         String selector = SDX_DELETE_EVENT.event();
         String userId = threadBasedUserCrnProvider.getUserCrn();
-        String requestId = threadBasedRequestIdProvider.getRequestId();
+        String requestId = MDCBuilder.getRequestId().get();
```
Comment (attilapalfi92) [inline datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java]: Is it a good approach to use the MDCBuilder for getting the request ID here? MDC has the purpose of building the MDC context, I think this is abusing it.

Comment (horadla23) [reply datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java]: ok, i'll create a util class to get it

**Comment by horadla23** [comment]
Tested with SDX creation (with database), requestId created in SDX service was found in environment, redbeams and cloudbreak.

**Comment by horadla23** [comment]
As discussed, I'll do some refactor/POC to propagate requestId into every flow

**Comment by horadla23** [comment]
Working: handlers, pollers in SDX
Need to check: flow actions, pollers in env, freeipa, etc.

**Comment by horadla23** [comment]
retest this please

**Pull request merged**